antispam_bee.php

```diff
@@ -2391,6 +2408,21 @@ public static function delete_spam_reason_by_comment( $comment_id ) {
 		);
 	}
 
+	/**
+	 * Updates the Antispam Bee reason for manual transitions
+	 *
+	 * @since   2.9.2
+	 * @param  [int] $comment Comment Object or comment ID.
+	 */
+	public static function update_antispam_bee_reason( $comment ) {
+		if ( ! is_object( $comment ) ) {
```

Comment:
should we check if `get_comment_meta($comment->ID, 'antispam_bee_reason', true)` has already a reason? If someone approves a spam comment and decides this was a mistake, the original reason would remain :thinking: 

But not quite sure, if that's a good idea.

Reply:
If the comment is marked as spam through any other reason and the metadata is set it can be rescued to approved/pending. If it is then marked as spam manually ASB was correct and the original reason was correct, so yes, this makes sense.

What do other think?

Reply:
This does not work at the moment, because we intentionally delete the reason on unspam.
https://github.com/pluginkollektiv/antispam-bee/blob/8f3e23911a0d1ab56b94cd6dbe93c363ca827106/antispam_bee.php#L98-L104

Reply:
Thanks for looking into it. So we leave it as it is.